### PR TITLE
Include Tilt as a dependency in .gemspec

### PR DESCRIPTION
**Pull request by sidonath**
The latest version is specified, but this may need to be adjusted

### CODE REVIEW

**Comment by dblock** [comment]
+1, I just got hit by this on Heroku :)

**Comment by hron** [comment]
+1. This affects me too. Please merge.

**Comment by dgilperez** [comment]
+1

**Comment by tschmidt** [comment]
Ran into the same issue as well.

File: mail_view.gemspec [hunk 1]
```diff
@@ -6,5 +6,7 @@ Gem::Specification.new do |s|
   s.summary = 'Visual email testing'
   s.homepage = 'https://github.com/37signals/mail_view'
 
-  s.files = Dir["#{File.dirname(__FILE__)}/*/**"]
+  s.add_dependency 'tilt', ['>= 1.3.3']
+
+  s.files = Dir["#{File.dirname(__FILE__)}/*/**/*"]
```
Comment (jeremy) [inline mail_view.gemspec:11]: This is unnecessary and unrelated

**Comment by jeremy** [comment]
Tilt dep added in a99c2fc242fbb271a739a0934c59d4d66d3cf9c7

**Comment by tschmidt** [comment]
Thanks @jeremy

**Pull request closed without merging**